base/bitarray.jl

```diff
@@ -1452,22 +1456,17 @@ circshift!(B::BitVector, i::Integer) = circshift!(B, B, i)
 
 ## count & find ##
 
-function count(B::BitArray)
+function bitcount(Bc::Vector{UInt64})
     n = 0
-    Bc = B.chunks
     @inbounds for i = 1:length(Bc)
         n += count_ones(Bc[i])
     end
     return n
 end
 
-# returns the index of the next non-zero element, or 0 if all zeros
-function findnext(B::BitArray, start::Integer)
-    start > 0 || throw(BoundsError(B, start))
-    start > length(B) && return 0
-
-    Bc = B.chunks
+bitcount(B::BitArray) = bitcount(B.chunks)
```

Comment:
`count(B::BitArray)`?

Reply:
Or even `sum` although I think that's the same thing when the eltype is Bool.